src/site/content/en/blog/catalina-system-ui/index.md

```diff
@@ -0,0 +1,246 @@
+---
+title: More variable font options for the macOS `system-ui` font in Chromium 83
+subhead: Catalina brings a new united variable system font to macOS 
+authors:
+  - adamargyle
+  - drott
+description: Chromium 80 had a system-ui font weight regression on macOS. The reason it broke, and the new super powers post-resolution, are worth the wait in Chromium 83.
+date: 2020-05-21
+updated: 2020-05-21
+hero: hero.jpg
+thumbnail: thumb.png
+alt: A bright pink and purple gradient with "macOS Catalina system-ui" going from thin to think from left to right, demonstrating some of the new variation settings
+tags:
+  - blog
+  - css
+  - fonts
+---
+
+The ['system-ui' section](https://drafts.csswg.org/css-fonts-4/#system-ui-def) of the CSS Fonts Module Level 4 spec defines a `system-ui` font keyword that allows developers to use the built-in, turbo-optimized, localized, mega-high-quality, no-download-needed, default operating system font font right in their sites and apps. 
+
+```css
+body {
+  font-family: system-ui;
+}
+```
+
+This typography choice is akin to saying:
+
+{% Blockquote 'browser' %}
+use the native system font for the current locale of this user
+{% endBlockquote %}
```

Comment:
```suggestion
This typography choice is akin to saying "use the native system font for the current locale of this user."
```
This is formatting weird so let's just use normal quotes.

![image](https://user-images.githubusercontent.com/4713486/82603358-5f3d2180-9b67-11ea-9764-4d1642c4389d.png)